source/physics/src/GateVSourceMessenger.cc

```diff
@@ -253,20 +247,18 @@ void GateVSourceMessenger::SetNewValue(G4UIcommand* command,G4String newValue)
       m_source->GetEneDist()->SetMinEnergy(setMinEnergycmd->GetNewDoubleValue(newValue));
   } else if( command == VisualizeCmd ) {
     m_source->Visualize(newValue);
-  }
-  else if( command == setEnergyRangecmd ) {
+  } else if( command == setEnergyRangecmd ) {
       m_source->GetEneDist()->SetEnergyRange(setEnergyRangecmd->GetNewDoubleValue(newValue));
+  } else if( command == LinearPolarizationCmd) {
+	  G4double vl;
+	  char unts[30];
```

Comment:
is this guaranteed to have 30 elementes

Reply:
It's relict code just for test - here I can use function which dont care abount units.

Reply:
ok, would be better